zeppelin-web/src/app/interpreter/interpreter.controller.js

```diff
@@ -26,6 +26,7 @@ angular.module('zeppelinWebApp').controller('InterpreterCtrl',
     var getInterpreterSettings = function() {
       $http.get(baseUrlSrv.getRestApiBase() + '/interpreter/setting').success(function(data, status, headers, config) {
         $scope.interpreterSettings = data.body;
+        checkDownloadingDependencies();
```

Comment:
Have added another condition that will check for route url, and if it is not `/interpreter` stop requesting for getInterpreterSettings().